packages/alea-frontend/pages/blog/new.tsx

```diff
@@ -26,17 +27,25 @@ const NewPostPage: NextPage = () => {
   const [body, setBody] = useState('');
   const postId = generatePostId(title);
 
+  if (title) localStore.setItem('blogTitle', title);
+  if (body) localStore.setItem('blogBody', body);
```

Comment:
rename to `draft-blogTitle` and `draft-blogBody` and create constants for these at the top of this file.